### DISCUSSION TITLE
Pc crashes when injecting packages

### DISCUSSION THREAD

**Discussion by ze-nite** [aircrack-ng/aircrack-ng#2479] [Q&A]
this the syslog until the crash

My wirelles card is RTL8852AE and it supports both monitor mode and packet injection

`fev 28 17:34:26 diogo NetworkManager[789]: <info>  [1677605666.6486] device (p2p-dev-wlan0): state change: disconnected -> unmanaged (reason 'removed', sys-iface-state: 'removed')
fev 28 17:34:26 diogo NetworkManager[789]: <info>  [1677605666.6540] exiting (success)
fev 28 17:34:26 diogo systemd[1]: NetworkManager.service: Deactivated successfully.
fev 28 17:34:26 diogo systemd[1]: Stopped NetworkManager.service - Network Manager.
fev 28 17:34:26 diogo wpa_supplicant[5861]: wlan0: CTRL-EVENT-DSCP-POLICY clear_all
fev 28 17:34:26 diogo wpa_supplicant[5861]: nl80211: deinit ifname=wlan0 disabled_11b_rates=0
fev 28 17:34:26 diogo sudo[6705]: pam_unix(sudo:session): session closed for user root
fev 28 17:34:26 diogo systemd[1]: wpa_supplicant.service: Main process exited, code=killed, status=9/KILL
fev 28 17:34:26 diogo systemd[1]: wpa_supplicant.service: Failed with result 'signal'.
fev 28 17:34:30 diogo sudo[6847]:    diogo : TTY=pts/0 ; PWD=/home/diogo ; USER=root ; COMMAND=/usr/sbin/airmon-ng check kill
fev 28 17:34:30 diogo sudo[6847]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
fev 28 17:34:30 diogo sudo[6847]: pam_unix(sudo:session): session closed for user root
fev 28 17:34:36 diogo systemd[1]: NetworkManager-dispatcher.service: Deactivated successfully.
fev 28 17:34:36 diogo sudo[6987]:    diogo : TTY=pts/0 ; PWD=/home/diogo ; USER=root ; COMMAND=/usr/sbin/airmon-ng start wlan0
fev 28 17:34:36 diogo sudo[6987]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
fev 28 17:34:36 diogo kernel: rtw89_8852ae 0000:03:00.0: MAC has already powered on
fev 28 17:34:36 diogo kernel: IPv6: ADDRCONF(NETDEV_CHANGE): wlan0: link becomes ready
fev 28 17:34:36 diogo sudo[6987]: pam_unix(sudo:session): session closed for user root
fev 28 17:35:01 diogo CRON[7285]: pam_unix(cron:session): session opened for user root(uid=0) by (uid=0)
fev 28 17:35:01 diogo CRON[7286]: (root) CMD (command -v debian-sa1 > /dev/null && debian-sa1 1 1)
fev 28 17:35:01 diogo CRON[7285]: pam_unix(cron:session): session closed for user root
fev 28 17:35:19 diogo sudo[7445]:    diogo : TTY=pts/0 ; PWD=/home/diogo ; USER=root ; COMMAND=/usr/sbin/iw dev
fev 28 17:35:19 diogo sudo[7445]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
fev 28 17:35:19 diogo sudo[7445]: pam_unix(sudo:session): session closed for user root
fev 28 17:35:38 diogo sudo[7614]:    diogo : TTY=pts/0 ; PWD=/home/diogo ; USER=root ; COMMAND=/usr/bin/wash -i wlan0
fev 28 17:35:38 diogo sudo[7614]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
fev 28 17:35:38 diogo kernel: device wlan0 entered promiscuous mode
fev 28 17:35:58 diogo kernel: device wlan0 left promiscuous mode
fev 28 17:35:58 diogo sudo[7614]: pam_unix(sudo:session): session closed for user root
fev 28 17:38:07 diogo sudo[8842]:    diogo : TTY=pts/0 ; PWD=/home/diogo ; USER=root ; COMMAND=/usr/bin/reaver -i wlan0 -b 58:FC:20:BD:92:60 -K
fev 28 17:38:07 diogo sudo[8842]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
fev 28 17:38:07 diogo kernel: device wlan0 entered promiscuous mode`

**Comment by ze-nite** [comment]
its easier to read this way

fev 28 17:34:26 diogo NetworkManager[789]: <info>  [1677605666.6486] device (p2p-dev-wlan0): state change: disconnected -> unmanaged (reason 'removed', sys-iface-state: 'removed')
fev 28 17:34:26 diogo NetworkManager[789]: <info>  [1677605666.6540] exiting (success)
fev 28 17:34:26 diogo systemd[1]: NetworkManager.service: Deactivated successfully.
fev 28 17:34:26 diogo systemd[1]: Stopped NetworkManager.service - Network Manager.
fev 28 17:34:26 diogo wpa_supplicant[5861]: wlan0: CTRL-EVENT-DSCP-POLICY clear_all
fev 28 17:34:26 diogo wpa_supplicant[5861]: nl80211: deinit ifname=wlan0 disabled_11b_rates=0
fev 28 17:34:26 diogo sudo[6705]: pam_unix(sudo:session): session closed for user root
fev 28 17:34:26 diogo systemd[1]: wpa_supplicant.service: Main process exited, code=killed, status=9/KILL
fev 28 17:34:26 diogo systemd[1]: wpa_supplicant.service: Failed with result 'signal'.
fev 28 17:34:30 diogo sudo[6847]:    diogo : TTY=pts/0 ; PWD=/home/diogo ; USER=root ; COMMAND=/usr/sbin/airmon-ng check kill
fev 28 17:34:30 diogo sudo[6847]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
fev 28 17:34:30 diogo sudo[6847]: pam_unix(sudo:session): session closed for user root
fev 28 17:34:36 diogo systemd[1]: NetworkManager-dispatcher.service: Deactivated successfully.
fev 28 17:34:36 diogo sudo[6987]:    diogo : TTY=pts/0 ; PWD=/home/diogo ; USER=root ; COMMAND=/usr/sbin/airmon-ng start wlan0
fev 28 17:34:36 diogo sudo[6987]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
fev 28 17:34:36 diogo kernel: rtw89_8852ae 0000:03:00.0: MAC has already powered on
fev 28 17:34:36 diogo kernel: IPv6: ADDRCONF(NETDEV_CHANGE): wlan0: link becomes ready
fev 28 17:34:36 diogo sudo[6987]: pam_unix(sudo:session): session closed for user root
fev 28 17:35:01 diogo CRON[7285]: pam_unix(cron:session): session opened for user root(uid=0) by (uid=0)
fev 28 17:35:01 diogo CRON[7286]: (root) CMD (command -v debian-sa1 > /dev/null && debian-sa1 1 1)
fev 28 17:35:01 diogo CRON[7285]: pam_unix(cron:session): session closed for user root
fev 28 17:35:19 diogo sudo[7445]:    diogo : TTY=pts/0 ; PWD=/home/diogo ; USER=root ; COMMAND=/usr/sbin/iw dev
fev 28 17:35:19 diogo sudo[7445]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
fev 28 17:35:19 diogo sudo[7445]: pam_unix(sudo:session): session closed for user root
fev 28 17:35:38 diogo sudo[7614]:    diogo : TTY=pts/0 ; PWD=/home/diogo ; USER=root ; COMMAND=/usr/bin/wash -i wlan0
fev 28 17:35:38 diogo sudo[7614]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
fev 28 17:35:38 diogo kernel: device wlan0 entered promiscuous mode
fev 28 17:35:58 diogo kernel: device wlan0 left promiscuous mode
fev 28 17:35:58 diogo sudo[7614]: pam_unix(sudo:session): session closed for user root
fev 28 17:38:07 diogo sudo[8842]:    diogo : TTY=pts/0 ; PWD=/home/diogo ; USER=root ; COMMAND=/usr/bin/reaver -i wlan0 -b 58:FC:20:BD:92:60 -K
fev 28 17:38:07 diogo sudo[8842]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
fev 28 17:38:07 diogo kernel: device wlan0 entered promiscuous mode
-------------------------------------------------------------------------------------------------------------------------------------------------------------

**Comment by Mister-X-** [comment]
It's pretty useless, but it looks like that you may have forgotten to kill the network manager before putting it in monitor mode.

It's possible the driver is unstable but since you don't mention kernel, driver, and adapter chipset, there is no way for us to tell what's wrong.

**Reply by ze-nite** [reply]
> It's pretty useless, but it looks like that you may have forgotten to kill the network manager before putting it in monitor mode.
> 
> It's possible the driver is unstable but since you don't mention kernel, driver, and adapter chipset, there is no way for us to tell what's wrong.

More system info:

- Linux diogo 6.1.0-kali5-amd64 SMP PREEMPT_DYNAMIC Debian 6.1.12-1kali2 (2023-02-23) x86_64 GNU/Linux

![Captura de ecrã_2023-02-28_19-35-44](https://user-images.githubusercontent.com/33848154/221960226-b95b3640-2aee-4a5a-84ff-aa629f9b0380.png)

![Captura de ecrã_2023-02-28_19-36-07](https://user-images.githubusercontent.com/33848154/221960234-b0651f84-6b1f-4f5a-9e85-1d826cc55e85.png)

**Reply by Mister-X-** [reply]
`airmon-ng --verbose` was what I was expecting.

In any case, the driver indicates this is 802.11ax realtek, and I'm guessing it's just not stable at this time. Find a way to capture kernel panic and the logs surrounding it, then report back with it.

And as a tip in the future, not just for here, but anywhere, I suggest to format your posts, even if that takes a bit of time. Not doing so sounds like "I don't care", and you have a higher chance of your post being ignored. Providing useful information such as what is asked in the bug report will help more, and waste less time going back and forth asking for it.
